openapi/components/schemas/Applications/Application.yaml

```
@@ -74,7 +74,17 @@ properties:
       - GetInvoiceCollection
       - GetCustomer
       - GetCustomerCollection
-
+  configurationUrl:
+    description: External URL to visit in order to configure the application during installation.
```

suggestion: suggestion for readability 
```suggestion
    description: External URL where the application is configured during installation.
```